pages/treeherder.py

```diff
@@ -51,6 +59,42 @@ def open_perfherder_page(self):
         from perfherder import PerfherderPage
         return PerfherderPage(self.base_url, self.selenium).wait_for_page_to_load()
 
+    def select_next_job(self):
+        el = self.selenium.find_element(*self._resultset_locator)
+        Wait(self.selenium, self.timeout).until(EC.visibility_of(el))
+        el.send_keys(Keys.ARROW_RIGHT)
+        Wait(self.selenium, self.timeout).until(lambda s: self.job_result_status)
+        return self.selenium.find_element(*self._next_job_title).get_attribute('title')
```

Comment:
I would separate selecting the next job from retrieving the title of the next job, rather than returning it here. It's also a little unclear because would this be the next job after selecting the next job, or the current job after selecting the next job?